src/main/java/com/github/joschi/jadconfig/JadConfig.java

```diff
@@ -2,6 +2,7 @@
 
 import com.github.joschi.jadconfig.converters.NoConverter;
 import com.github.joschi.jadconfig.converters.StringConverter;
+import org.checkerframework.checker.nullness.qual.Nullable;
```

Comment:
I think we should use the `javax.annotation.Nullable` import here.

Reply:
Thanks, fixed.